### PR TITLE
Added Table of Contents to README

### PR DESCRIPTION
**Pull request by Sustenance**
Since the README is fairly long, it would make it easier for readers to navigate to have a ToC.

### CODE REVIEW

**Comment by shaunwarman** [comment]
Thanks @Sustenance

File: README.md [hunk 1]
```diff
@@ -11,6 +11,26 @@ Lead Maintainer: [Jean-Charles Sisk](https://github.com/jasisk)
...
+  * [Middleware](#middleware)
+    * [Included Middleware](#included-middleware)
+    * [Extending Default Middleware](#extending-default-middleware)
+  * [Application Security](#application-security)
+  * [Lifecycle Events](#lifecycle-events)
+  * [Configuration-based express Settings](#configuration-based-express-settings)
+  * [View Engine Configuration](#view-engine-configuration)
+* [Tests](#tests)
+* [Coverage](#coverage)
+* [Reading app configs from within the kraken app](#reading-app-configs-from-within-the-kraken-app)
```
Comment (shaunwarman) [inline README.md:33]: Looks like this may not correctly resolve.

Comment (Sustenance) [reply README.md:33]: Fixed. The code block in the `Coverage` section was broke.

**Comment by shaunwarman** [comment]
👍

**Comment by grawk** [comment]
Looks good. Thanks! 👍

**Pull request merged**